scripts/deployment/local.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# To be executed from the root project directory.
+# Installs R package locally and copies shiny resources from package 'shiny' directory to web application.
+# Input parameters:
+# -s [required] shiny webapp location where all shiny resources will be deployed
+# -p [optional] app package location, not required when build script was executed and both 'latest.txt' and package itself are available in 'target' directory
+
+while getopts ":s:p:" opt; do
+  case $opt in
+    s) shiny_webapp_location="$OPTARG"
+    ;;
+    p) package_location="$OPTARG"
+    ;;
+    \?) echo "Invalid option -$OPTARG" >&2
+    ;;
+  esac
+done
+
+#quitting on error
+set -e
+
+if [ -z "${shiny_webapp_location}" ]; then
+    echo -s input parameter value with shiny webapp location was not provided!
+    exit 1
+fi
+
+if [ -z "${package_location}" ]; then
+    latest_txt_file_location='target/latest.txt'
+    if [ -f "${latest_txt_file_location}" ]; then
+	package_location=`pwd`/target/`cat ${latest_txt_file_location}`
+    else
+	echo ${latest_txt_file_location} file not found!
+	exit 1
+    fi
+fi
+
+echo got package_location parameter: ${package_location}
+echo got shiny_webapp_location parameter: ${shiny_webapp_location}
+
+package_name=`tar -ztf ${package_location} | grep -o '^[^/]\+' | sort -u`
+
+tmp_for_shiny=/tmp/${package_name}_shiny_extr
+
+#installing module in local system
+R CMD INSTALL ${package_location}
```

Comment:
This should line doesn't fit here since it separates two code blocks (line 42 and lines 49-52) that deal with the same thing. I would move it to line 39.